tfhe/src/shortint/server_key/tests/noise_level.rs

```diff
@@ -254,6 +254,7 @@ fn test_ct_scalar_op_assign_noise_level_propagation(sk: &ServerKey, ct: &Ciphert
 }
 
 #[test]
+#[cfg(not(tarpaulin))] // This test is ignored in coverage, it takes around 4 hours to run otherwise.
```

Comment:
question should we use cfg(not(tarpaulin)) or cfg(not(__coverage)) ?

Reply:
We should select the second option indeed, just for the sake of being consistent with the rest of the `cfg` directives related to coverage.